public/account.html

```diff
@@ -83,7 +83,9 @@
 
           <p>
             <button class="primary" type="submit">Send</button>
-            <a href="#" data-action="show-signin">Cancel</a>
+            <small>
```

Comment:
we discussed it with Julia. Julia suggested to make an extra class for links like `cancel`, similar to Bootstrap’s `btn-link` class